docs/PIPELINES-CONFIG.md

```diff
@@ -0,0 +1,269 @@
+## Pipelines/Tekton Configuration
+
+As part of this section you will find information about the following configuration methods:
+
+1. [Configuring Pipelines/Tekton with the configuration script after using our `ai-rhdh-installer`](#ai-rhdh-installer-script-configuration)
+2. [Configuring Pipelines/Tekton manually after using our `ai-rhdh-installer`](#ai-rhdh-installer-manual-configuration)
+3. [Configuring Pipelines/Tekton with the configuration script for a pre-existing Red Hat Developer Hub instance](#pre-existing-instance-script-configuration)
+4. [Configuring Pipelines/Tekton manually for a pre-existing Red Hat Developer Hub instance](#pre-existing-instance-manual-configuration)
+
+### Prerequisites
+
+- [yq](https://github.com/mikefarah/yq/) version 4.0+.
+- [kubectl](https://github.com/kubernetes/kubectl) version compatible with your target cluster
```

Comment:
I think `kubectl` should also be included in the `GITOPS-CONFIG.md` pre-requisites section. Might be better to mention that having `oc` installed is also ok?

Reply:
If we add this we should add the requirement to gitops docs as well if it isn't already there

Reply:
Done in https://github.com/redhat-ai-dev/ai-rhdh-installer/pull/7/commits/17c441450331fba50a78a680b3cfc238a4275a47